x/validator-preference/README.md

```diff
@@ -0,0 +1,155 @@
+# validator-set Preference 
+
+## Abstract 
+
+Validator-Set preference is a new module which gives users and contracts a 
+better UX for staking to a set of validators. For example: a one click button
+that delegates to multiple validators. Then the user can set (or realistically a frontend provides) 
+a list of recommended defaults (Ex: governors, wosmongton, chain/stack contributors etc).
+Currently this can be done on-chain with frontends, but having a preference list stored locally 
+eases frontend code burden. 
```

Comment:
```suggestion
Currently this can be done on-chain with frontends, but having a preference list stored on-chain 
lowers front-end code burden and eliminates potential front-end lock in.
```